Setting for rabbitmq server address

Galexrt opened the pull request:
Added setting for rabbitmq server address.
The "default" is `localhost`.
The variable name is `RABBITMQ_HOST`.

smarx commented:
*Automated message from Dropbox CLA bot*

@Galexrt, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/) and update the thread so we can consider merging your code.

Galexrt commented:
@smarx I signed the CLA.
(The second time now, maybe I'm doing something wrong, if so what?)

Galexrt commented:
The build fail, doesn't seem to be related to my changes.

***

```
Errors were encountered while processing:
 postgresql-9.4
 postgresql-contrib-9.4
E: Sub-process /usr/bin/dpkg returned an error code (1)
The command "./tools/travis/$TEST_SUITE" exited with 100.
Done. Your build exited with 1.
```
See https://travis-ci.org/zulip/zulip/jobs/103058113

timabbott commented:
Yeah that seems to be a regression caused by Travis CI changing something.  They aren't the best provider.  I'll track it down when I get a chance.  

I think for the option, we want to support setting the rabbitmq host in /etc/zulip/settings.py aka zproject/local_settings_template.py , so I think you'll want to make the default of localhost set in  DEFAULT_SETTINGS maybe with a comment in the RABBITMQ section.

Galexrt commented:
@timabbott I moved the `RABBITMQ_HOST` to the `DEFAULT_SETTINGS and added a comment to the RABBITMQ Section.
I also added the var commented in the `local_settings.py`, for new users to see it while editing the settings. Is this how it should be changed?

timabbott commented on zproject/local_settings_template.py:
```diff
@@ -298,3 +298,6 @@
 
 CAMO_URI = ''
+
+# Set the rabbitmq host(s)
```
Would probably expand this comment a bit -- "By default, Zulip connects to rabbitmq running locally on the machine, but Zulip also supports connecting to RabbitMQ over the network; to use a remote RabbitMQ instance, set RABBITMQ_HOST here."

timabbott commented:
Posted one quick comment and you should squash it into a single commit, otherwise looks good to me.

Galexrt commented:
@timabbott Changed the comment in the file to your comment and squashed everything into one commit.

Galexrt commented:
@timabbott I just created a new branch with `RABBITMQ_USERNAME`, `REDIS_*` and the `MEMCACHED_LOCATION` configurable. Should I create a new pull request for those changes or add the changes to this pull request?
